README.md

```diff
@@ -32,6 +32,14 @@ $copyleaks = new Copyleaks();
 $loginResult = $copyleaks->login(<your email>,<your api key>);
 echo json_encode($loginResult);
 ```
+* (Option) To change the Identity server URI (default:"https://id.copyleaks.com"):
+```rb
+CopyleaksConfig::SET_IDENTITY_SERVER_URI("<your identity server URI>"); 
```

Comment:
Unable to see this function in the changes

Reply:
The setters for the identity and API server URI were already exist at the config file.